training/src/training/bugReport1

```diff
@@ -0,0 +1,6 @@
+Bug Report #1
```

Comment:
Make your steps more verbose. They should be in a numbered list (1,2,3,etc).
Include an expected result for each step.
Since this is a web app you can include the browser version and type.
In practice we can include traceability to the feature or the testcase.
Add screenshots, videos, etc.
Include Severity & Priority

https://testlio.com/blog/the-ideal-bug-report/